app/assets/javascripts/controllers/cloud_network/cloud_network_form_controller.js

```diff
@@ -1,4 +1,4 @@
-ManageIQ.angular.app.controller('cloudNetworkFormController', ['cloudNetworkFormId', 'miqService', 'API', function(cloudNetworkFormId, miqService, API) {
+ManageIQ.angular.app.controller('cloudNetworkFormController', ['cloudNetworkFormId', 'networkProviderChoices', 'miqService', 'API', function(cloudNetworkFormId, miqService, API) {
```

Comment:
`function(cloudNetworkFormId, miqService, API)` is misssing `networkProviderChoices`.